samples/IISSample/project.json

```diff
@@ -23,5 +23,11 @@
     "include": [
       "web.config"
     ]
+  },
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-preview2-final"
```

Comment:
This one is awkward because the tool no longer exists. @natemcmaster we should keep using this version right?